User_Story#1/SortingApp/BusinessLayer/Utilities/Parser/Interfaces/IDataParser.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace BusinessLayer.Utilities.Parser.Interfaces
+{
+    /// <summary>
+    /// Provides functionality that parses data.
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    public interface IDataParser<T> where T : IComparable
+    {
+        /// <summary>
+        /// Converts data to collection of srecific type.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// 
+        T[] DataConverter(string content);
```

Comment:
Method should be named as a verb.

Reply:
Also don't forget about parameter name.

Reply:
Parameter name changed in Validator class, there I really check path instead of content.